com.unity.render-pipelines.universal/Documentation~/urp-shaders/urp-shaderlab-pass-tags.md

```diff
@@ -2,23 +2,37 @@
 | :--- | :--- |
 | **UniversalForward** | The Pass renders object geometry and evaluates all light contributions. URP uses this tag value in the Forward Rendering Path. |
-| **UniversalGBuffer** | The Pass renders object geometry without evaluating any light contribution. URP uses this tag value in the Deferred Rendering Path. |
-| **UniversalForwardOnly** | The Pass renders object geometry and evaluates all light contributions, similarly to when **LightMode** has the **UniversalForward** value. The difference from **UniversalForward** is that URP can use the Pass for both the Forward and the Deferred Rendering Paths.<br/>Use this value if a certain Pass must render objects with the Forward Rendering Path when URP is using the Deferred Rendering Path. For example, use this tag if URP renders a Scene using the Deferred Rendering Path and the Scene contains objects with shader data that does not fit the GBuffer, such as Clear Coat normals.<br/>If a shader must render in both the Forward and the  Deferred Rendering Paths, declare two Passes with the `UniversalForward` and `UniversalGBuffer` tag values. If a shader must render using the Forward Rendering Path regardless of the Rendering Path that the URP Renderer uses, declare only a Pass with the `LightMode` tag set to `UniversalForwardOnly`. |
+| **UniversalGBuffer** | The Pass renders object geometry without evaluating any light contribution. Use this tag value in Passes that Unity must execute in the Deferred Rendering Path. |
+| **UniversalForwardOnly** | The Pass renders object geometry and evaluates all light contributions, similarly to when **LightMode** has the **UniversalForward** value. The difference from **UniversalForward** is that URP can use the Pass for both the Forward and the Deferred Rendering Paths.<br/>Use this value if a certain Pass must render objects with the Forward Rendering Path when URP is using the Deferred Rendering Path. For example, use this tag if URP renders a Scene using the Deferred Rendering Path and the Scene contains objects with shader data that does not fit the GBuffer, such as Clear Coat normals.<br/>If a shader must render in both the Forward and the  Deferred Rendering Paths, declare two Passes with the `UniversalForward` and `UniversalGBuffer` tag values.<br/>If a shader must render using the Forward Rendering Path regardless of the Rendering Path that the URP Renderer uses, declare only a Pass with the `LightMode` tag set to `UniversalForwardOnly`.<br/>If you use the SSAO Renderer Feature, add a Pass with the `LightMode` tag set to `DepthNormalsOnly`. For more information, see the `DepthNormalsOnly` value. |
+| **DepthNormalsOnly** | Use this value in combination with `UniversalForwardOnly` in the Deferred Rendering Path. This value lets Unity render the shader in the Depth and normal prepass. In the Deferred Rendering Path, if the Pass with the `DepthNormalsOnly` tag value is missing, Unity does not generate the ambient occlusion around the Mesh. |
```

Comment:
Is this specific to deferred though? Won't a depth + normal be used for something like Forward DBuffer when using a prepass? @lukaschod @kaychang-unity

Reply:
Forward renderer will use "DepthNormals" for its depth-normal prepass.
Deferred renderer will use "DepthNormalsOnly" for its depth-normal prepass.
The different (confusing) names is needed to differentiate between standard materials than can be rendered in both deferred or forward rendering mode, and the forward-only materials.
I DM-ed @oleks-k to add the missing section about "DepthNormals" pass.

Reply:
Should the name be `DepthNormalsForwardOnly` to be more clear?

Reply:
It could be.
A separate PR will be needed to amend the code.

Reply:
Thank you, I've added this to a separate task:
https://unity3d.atlassian.net/browse/DOC-3262